src/plugins/unified_search/public/filter_bar/filter_editor/index.tsx

```diff
@@ -84,16 +85,30 @@ class FilterEditorUI extends Component<Props, State> {
   }
 
   public render() {
+    // TODO: Hook up "Add vs Edit" panel titles
+    const panelTitleAdd = i18n.translate('unifiedSearch.filter.filterEditor.addFilterPopupTitle', {
+      defaultMessage: 'Add filter',
+    });
+    const panelTitleEdit = i18n.translate(
+      'unifiedSearch.filter.filterEditor.editFilterPopupTitle',
+      {
+        defaultMessage: 'Edit filter',
+      }
+    );
```

Comment:
Can you help with this TODO? We need to change the popover title from "Edit" to "Add" when in adding mode.

Reply:
Done https://github.com/elastic/kibana/pull/129877/commits/874133016587e680ac461bfa1799663888e906ff